dist/common_functions.py

```diff
@@ -0,0 +1,22 @@
+#! /bin/python3
```

Comment:
Would prefer no space, but our style is inconsistent so up to you.

Reply:
Changed in [71091ae](https://github.com/wiredtiger/wiredtiger/pull/9397/commits/71091ae8db66d91106a3abc679c1ca9d63b13782)